dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ResolveDirector.java

```diff
@@ -417,6 +429,12 @@ void resolve(String feature, File esaFile, String toExtension) throws InstallExc
 
         RepositoryResolver resolver;
         Collection<List<RepositoryResource>> installResources;
+        String edition = this.product.getProductEdition();
```

Comment:
As in other comments, use getProductId() to check for io.openliberty.  Don't need the "Open Liberty" or " edition." strings.